minerva/whales/preprocessing.py

```diff
@@ -96,33 +99,51 @@ def __init__(self, X, y, img_dirpath, augmentation, target_size, bins_nr):
         self.preprocessing_function = None
         self.normalization_function = None
 
-    def load_image(self, img_name):
-        img_filepath = os.path.join(self.img_dirpath, img_name)
-        return Image.open(img_filepath, 'r')
+    def load_image(self, img_name, max_scale=8):
+        import turbojpeg
+        img_path = Path(self.img_dirpath) / img_name
+        img = Image.open(img_path.as_posix())
+
+        scale = 0.5
```

Comment:
@pawel-ziecina what is the difference between `max_scale=8` and `scale = 0.5`. Do you think that we should parametrize this -> move it to config file?

(@jakubczakon )